tools/telemetry-generator/test/appInsightsExecutionTimeTestHandler.test.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { TelemetryClient } from "applicationinsights";
+
+const handler = require("../src/handlers/appInsightsExecutionTimeTestHandler");
+
+describe("appInsightsExecutionTimeTestHandler", () => {
+    let mockTelemetryClient: TelemetryClient;
+    let mockFileData: any;
+
+    beforeEach(() => {
+        mockTelemetryClient = {
+            trackMetric: jest.fn(),
+        } as unknown as TelemetryClient;
+
+        mockFileData = {
```

Comment:
If this is only used for the one test, it probably should be defined inside that test instead of the beforeEach hook.